Please accept my pull request for Maths repo. @ Pull Request #1

dprasadk opened the pull request:
```
I have completed creating beginners content for Linear Algebra and Calculus , please accept.


Kind Regards,
dprasad
```

dprasadk committed:
@dprasad_commit-01

dprasadk committed:
Updated docs for Beginner Level Linear Algebra and Calculus #1

dprasadk committed:
Updated docs for Beginner Level Linear Algebra and Calculus #2

co-learning-lounge commented on Technology/Artificial Intelligence/Maths/prerequisite_for_AI/beginner/linear_algebra/README.md:
```diff
@@ -0,0 +1,48 @@
+- [A comprehensive beginners guide to Linear Algebra for Data Scientists]( https://www.analyticsvidhya.com/blog/2017/05/comprehensive-guide-to-linear-algebra/)
+- [Boost your data science skills. Learn linear algebra]( https://towardsdatascience.com/boost-your-data-sciences-skills-learn-linear-algebra-2c30fdd008cf)
+
+>***Stay tuned  - > Linear-Algebra-for-Intermediate !***
```
Remove this line.

co-learning-lounge commented on Technology/Artificial Intelligence/Maths/prerequisite_for_AI/beginner/linear_algebra/README.md:
```diff
@@ -0,0 +1,48 @@
+
+# Linear-Algebra-for-Beginners
```
Remove - between two words.

co-learning-lounge commented on Technology/Artificial Intelligence/Maths/prerequisite_for_AI/beginner/calculus/README.md:
```diff
@@ -0,0 +1,50 @@
+-  [Calculus 1 - A Must Know Concept For Every Professional](https://medium.com/fintechexplained/calculus-1-a-must-know-concept-for-every-professional-debd9c14f476)
+-  [Calculus 2 - A Must Know Concept For Every Professional](https://medium.com/fintechexplained/calculus-2-a-must-know-concept-for-every-professional-a86599caa2a5)
+
+>***Stay tuned - > Calculus-for-Intermediate !***
```
Remove this line.

co-learning-lounge commented on Technology/Artificial Intelligence/Maths/prerequisite_for_AI/beginner/calculus/README.md:
```diff
@@ -0,0 +1,50 @@
+# Calculus-for-Beginners
```
Remove - between two words.

co-learning-lounge commented on Technology/Artificial Intelligence/Maths/prerequisite_for_AI/beginner/README.md:
```diff
@@ -0,0 +1,18 @@
+> 
+> 
+> 
+> #### 1. Linear Algebra
```
Add hyperlink for the folders you created.

Also. Instead of writing with _ write like Prerequisite for AI or Linear Algebra

co-learning-lounge commented on Technology/Artificial Intelligence/Maths/prerequisite_for_AI/beginner/README.md:
```diff
@@ -0,0 +1,18 @@
+> 
+> 
+> #### 1. Linear Algebra
+> #### 2. Calculus
```
Add hyperlink for the folders you created.

co-learning-lounge commented on Technology/Artificial Intelligence/Maths/prerequisite_for_AI/beginner/README.md:
```diff
@@ -0,0 +1,18 @@
+
```
Is it possible to add more details? Like https://github.com/colearninglounge/co-learning-lounge/tree/master/Technology/Artificial%20Intelligence

co-learning-lounge reviewed (changes requested):
Reviewed and mentioned the required changes.

dprasadk committed:
@dprasadk Updated docs for Beginner Level Linear Algebra #2

Corrected added extra contents too, will add some intermediate contents in Intermediate. Please accept my changes.

dprasadk committed:
@dprasadk Updated docs for Beginner Level  Calculus #2

Please accept my reviewed changes.

dprasadk committed:
Updated docs for Beginner Level Linear Algebra and Calculus #3

dprasadk committed:
Updated README.md for  Prerequisite for AI Branch

Please accept.

co-learning-lounge commented:
Added Maths section inside AI technology.
